views/help/videos.html.twig

```diff
@@ -9,8 +9,55 @@
     <div class="page-header">
         <h2><i class="fa fa-film" aria-hidden="true"></i> Training Videos</h2>
     </div>
-    <div class="alert alert-warning lead">
-        Coming soon
+
+    {% set firstVideo = false %}
+    <div class="row">
+        <div class="col-sm-8 col-sm-push-4">
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <h3 class="panel-title" id="video-title"></h3>
+                </div>
+                <div class="panel-body">
+                    <div class="embed-responsive embed-responsive-16by9">
+                        <iframe id="video-embed" class="embed-responsive-item" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="col-sm-4 col-sm-pull-8">
+            {% for group in videoGroups %}
+                <h4>{{ group.title }}</h4>
+                <div class="list-group">
+                    {% for chapter, video in group.videos %}
+                        {% if video.link is not empty %}
+                            <a href="#" class="list-group-item load-video" data-embed-src="{{ video.link }}">{{ chapter }}: {{ video.title }}</a>
+                            {% if not firstVideo %}
+                                {% set firstVideo = video.link %}
+                            {% endif %}
+                        {% else %}
+                            <div class="list-group-item disabled">{{ chapter }}: {{ video.title }} <em><small>(Coming Soon!)</small></em></div>
+                        {% endif %}
+                    {% endfor %}
+                </div>
+                <br />
+            {% endfor %}
+        </div>
     </div>
     {% include 'help/footer.html.twig' %}
 {% endblock %}
+
+{% block pagejs %}
+<script>
+$(document).ready(function() {
+    $('a.load-video').on('click', function() {
+        $('#video-title').html($(this).html());
+        $('iframe#video-embed').attr('src', $(this).data('embed-src') + '?modestbranding=1&rel=0');
+        $('html').scrollTop($('iframe#video-embed').offset().top);
```

Comment:
I switched to target the panel in [this commit](https://github.com/all-of-us/healthpro/pull/419/commits/4946f0060262d993fa87792b7adf509af8fb9e91) but I'm not sure how I feel about the animation.  I think I would rather have it pop up to the top of the video instead of having to wait for the scroll to animate.

Reply:
👍